src/Stratis.Bitcoin.Features.BlockStore/Models/ObjectByHashRequest.cs

```diff
@@ -0,0 +1,17 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Text;
+
+namespace Stratis.Bitcoin.Features.BlockStore.Models
+{
+    public abstract class RequestBase
+    {
+        public bool OutputJson { get; set; }
+    }
+    public class ObjectByHashRequest : RequestBase
```

Comment:
Not sure this will always return an object. Like if we have a `DELETE /api/block?hash=dddddddd`.